### ISSUE TITLE
[DropDownButton] NVDA disables arrow keys keyboard navigation shortcut

### ISSUE THREAD

**Issue by zstoyanova9** [telerik/kendo-angular#4518]
### Description

When the NVDA screen reader is turned on, the default arrow keys keyboard navigation shortcut stops working properly and the items in the popup of the DropDownButton cannot be navigated. 

### Steps To Reproduce

1. Open the following StackBlitz example - https://stackblitz.com/edit/angular-j6klficc 
2. Turn on the NVDA screen reader.
3. Focus the component and open its popup.
4. Try to navigate through the items using the arrow keys.

### Screenshots or video

https://github.com/user-attachments/assets/f55d1a4c-bf39-4339-96c2-f61b2a40e460

### Actual Behavior

The items in the DropDownButton popup cannot be navigated with the arrow keys when NVDA is turned on.

### Expected Behavior

The NVDA screen reader should not affect the default behavior of the arrow keys keyboard navigation shortcut of the DropDownButton.

### Browser

Chrome

### Browser version

latest

### OS type

MacOS

### Last working version of the Kendo UI for Angular package (if regression).

v16.0.0

**Comment by mbechev** [comment]
The issue is also valid for the Splitter component when focusing the resizer and trying to resize the panes using the arrows:

<img width="422" alt="Image" src="https://github.com/user-attachments/assets/0cd69b90-684f-4922-b0b3-65b3965e0b36" />

The panes cannot be resized using the arrow keys when NVDA is enabled.
